nf_core/__main__.py

```diff
@@ -500,10 +500,11 @@ def create_test_yml(ctx, tool, run_tests, output, force, no_prompts):
 @click.pass_context
 @click.argument("pipeline_dir", type=click.Path(exists=True), required=True, metavar="<pipeline/modules directory>")
 @click.option("-t", "--tool", type=str, metavar="<tool> or <tool/subtool>")
+@click.option("-k", "--key", type=str, metavar="<test>", multiple=True, help="Run only these lint tests")
```

Comment:
Could we perhaps show which lint tests are possible to choose from? It might clutter the help message though.

Reply:
I actually think that's a really good idea, because currently you kinda have to know how the tests are named in the code. For the typical user, this function is therefore rather useless. Showing a list would be nice! We could have this for the general `nf-core lint` as well. Have to think about this though, so maybe we'll add it as a separate PR and then can do it for both linters.

Reply:
I've thought about this for the general linting before but thought it was too many.

We have the web documentation for the main pipeline linting which has all of the test names. I guess this doesn't work for the module lint tests yet? We should extend it to do so if not, as we need docs for each modules king test really.

Reply:
Yes good point ... probably should make an issue for it